packages/dds/tree/src/feature-libraries/flex-tree/lazyNode.ts

```diff
@@ -192,7 +192,9 @@ export class LazyTreeNode<TSchema extends FlexTreeNodeSchema = FlexTreeNodeSchem
 				// Additionally this approach makes it possible for a user to take a FlexTree node, get its parent, check its schema, down cast based on that, then edit that detached field (ex: removing the node in it).
 				// This MIGHT work properly with existing merge resolution logic (it must keep client in sync and be unable to violate schema), but this either needs robust testing or to be explicitly banned (error before s3ending the op).
 				// Issues like replacing a node in the a removed sequenced then undoing the remove could easily violate schema if not everything works exactly right!
-				fieldSchema = FlexFieldSchema.create(FieldKinds.sequence, [Any]);
+				fieldSchema = FlexFieldSchema.create(FieldKinds.sequence, [
+					...this.context.schema.nodeSchema.values(),
```

Comment:
This comment should be updated now.
Are you at all worried about the cost of copying all node types here?

Reply:
Its not great, but I don't realistically know of any use case for a user walking up the parent of the root of detached nodes in a performance sensitive way, especially since I've never seen any app do anything with detached nodes at all. If needed this would be easy to optimize, but I don't think it would ever be worth the complexity.

Reply:
commend updated.